doajtest/mocks/response.py

```diff
@@ -0,0 +1,57 @@
+from doajtest.fixtures.article_crossref import CrossrefArticleFixtureFactory
```

Comment:
I have not played with it, but @Steven-Eardley might be able to help - there's already a requests mocking library that we have installed as a dependency, so it may be worth replacing this class with that, or at least wrapping it.

Reply:
@Steven-Eardley could you please point me in the right direction?

Reply:
I never got around to this before release, I can catch up on this later